_data/sidebar_doc.yml

```diff
@@ -462,6 +462,9 @@ entries:
         - title: View Version Details
           url: /view-version-details.html
 
+        - title: Upgrade Cluster Version
```

Comment:
"Version" is a commonly used term when dealing with OSS. Important to have this in here for SEO.